### PR TITLE
Mentions how hub needs to be configured

### PR DESCRIPTION
**Pull request by laurent22**
This is to address issue #3531

- [X] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/brew/blob/master/CONTRIBUTING.md) document?
- [X] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/brew/pulls) for the same change?
- [X] Have you added an explanation of what your changes do and why you'd like us to include them?
- [ ] Have you written new tests for your changes? [Here's an example](https://github.com/Homebrew/homebrew/pull/49031).
- [ ] Have you successfully run `brew tests` with your changes locally?

-----

### CODE REVIEW

File: Library/Homebrew/dev-cmd/bump-formula-pr.rb [hunk 1]
```diff
@@ -338,7 +338,7 @@ def bump_formula_pr
 
         if reply.to_s.include? "username:"
           formula.path.atomic_write(backup_file) unless ARGV.dry_run?
-          odie "You need to configure hub"
+          odie "Please authentify with hub (eg. by typing 'cd $(brew --repo) && hub issue') and try again."
```
Comment (MikeMcQuaid) [inline Library/Homebrew/dev-cmd/bump-formula-pr.rb]: How about: `configure hub by running 'hub -C $(brew --repo)' and try again.`

Comment (DomT4) [reply Library/Homebrew/dev-cmd/bump-formula-pr.rb]: That's not enough AFAIK. If you temporarily remove `~/.config/hub` and run that command it works fine.

Comment (ilovezfs) [reply Library/Homebrew/dev-cmd/bump-formula-pr.rb]: It's a shame hub doesn't have a `hub configure` command.

I think @MikeMcQuaid means
```
hub -C $(brew --repo) issue
```

Comment (DomT4) [reply Library/Homebrew/dev-cmd/bump-formula-pr.rb]: > It's a shame hub doesn't have a hub configure command.

Aye, I went looking for a `hub whoami` command or something but drew a blank.

> hub -C $(brew --repo) issue

👍 That would work.

**Comment by MikeMcQuaid** [comment]
Will merge as-is and tweak in a follow-up PR. Thanks @laurent22!

**Pull request merged**